### DISCUSSION TITLE
Suggestion to add paths to the features1.json scene in all app params

### DISCUSSION THREAD

**Discussion by mmmovania** [xelatihy/yocto-gl#1262] [Ideas]
A quick suggestion. The default params scene attrib points to scene.ply which does not exist. What I suggest is that the default path of scene should be set to the features1.json which is shipped with the repo. For e.g. if i were to run the yscene app as view command, I can simply pass the features1.json scene relative path as follows:
```
// convert params
struct view_params : trace_params {
  string scene   = "../../../tests/features1/features1.json";
  string output  = "../../../tests/features1/features1.png";
  string camname = "";
  bool   addsky  = false;
  string envname = "";
};
```
This will save a lot of hassle for beginners who would be able to see something on screen instead of an error on command prompt.

**Comment by xelatihy** [comment]
I see. It makes sense. I'll add a default scene in the next release.

**Reply by mmmovania** [reply]
Great thanks :) The library is awesome :)
